helm-charts/helm-api/values.yaml

```diff
@@ -68,7 +68,8 @@ helmapi:
     mongoCaName: mongodb-ca-cert
     mongoClientCertMountPath: /certs/mongodb-client
     mongoClientCertName: mongodb-client-cert
-nodeSelector: {}
+nodeSelector: 
```

Comment:
we should remove nodeSelector all together ..there have been 2 build break because of this